arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,498 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2016-07-12-preview",
+		"title": "AdvisorManagementClient",
+		"description": "REST APIs for Azure Advisor"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations": {
+			"post": {
+				"tags": ["GenerateRecommendations"],
+				"description": "Initiates the recommendation generation/computation process for a subscription. This is an asynchronous operation. The generated recommendations are stored in a cache in the Advisor service.",
+				"operationId": "GenerateRecommendations_GenerateRecommendations",
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionIdParameter"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"202": {
+						"description": "Accepted",
+						"headers": {
+							"Location": {
+								"description": "The URL where the status of the long-running operation can be checked.",
+								"type": "string"
+							},
+							"Retry-After": {
+								"description": "The delay that the client should use when checking for the status of the operation. The value is in seconds.",
+								"type": "string"
+							}
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GenerateRecommendations": {
+						"$ref": "../examples/GenerateRecommendations.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations/{operationId}": {
+			"get": {
+				"tags": ["GenerateRecommendations"],
+				"description": "Get generate recommendations status. As generate recommendation operation is an asynchronous operation. This is the method to get generate recommendation status.",
+				"operationId": "GenerateRecommendations_GetGenerateRecommendationsStatus",
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionIdParameter"
+				},
+				{
+					"name": "operationId",
+					"in": "path",
+					"description": "The operation Id.",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"$ref": "#/parameters/apiVersionParameter"
+				}],
+				"responses": {
+					"202": {
+						"description": "Generate recommendation hasn't finished."
+					},
+					"204": {
+						"description": "Generate recommendation has finished."
+					},
+					"404": {
+						"description": "The operationId cannot be found."
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"PollGenerateRecommendationsStatus": {
+						"$ref": "../examples/PollGenerateRecommendationsStatus.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/recommendations": {
+			"get": {
+				"tags": ["GetRecommendations"],
```

Comment:
it looks like the validation tool didn't run my latest change? I have already fixed all the validation errors. And the last commit has already passed model validation.